### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by Salvin-George**
Hi, @Mai-Vang51 can you review my code. Specifically i want some feedback on the project_management file.

### CODE REVIEW

File: prac_07/project_management.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Actual:
+"""
+import csv
+from datetime import datetime
+
+FILENAME = "projects.txt"
+from project import Project
+
+
+def main():
```
Comment (Mai-Vang51) [inline prac_07/project_management.py:13]: Might be nice to add a docstring here to briefly explain what program will do

File: prac_07/project_management.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            cost_estimate = float(input("Cost estimate: $"))
+            completion_percentage = int(input("Percent complete: "))
+            project = Project(name, start_date, priority, cost_estimate, completion_percentage)
+            projects.append(project)
+
+        else:
+            print("Invalid choice. Please try again.")
+
+        print(
+            "- (L)oad projects\n- (S)ave projects\n- (D)isplay projects\n- (F)ilter projects by date\n- (A)dd new project\n- (U)pdate project\n- (Q)uit")
```
Comment (Mai-Vang51) [inline prac_07/project_management.py:46]: Could you have put this into a constant to avoid DRY?

File: prac_07/project_management.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        print(
+            f"{i}. {project.name}, start: {project.start_date.strftime('%d/%m/%Y')}, priority {project.priority}, estimate: ${project.cost_estimate:.2f}, completion: {project.completion_percentage}%")
+
+
+def filter_projects_by_date(projects, date_str):
+    try:
+        date = datetime.strptime(date_str, '%d/%m/%Y')
+    except ValueError:
+        print("Invalid date format. Please use 'dd/mm/yyyy' format.")
+        return
```
Comment (Mai-Vang51) [inline prac_07/project_management.py:101]: Could this be a separate function?

**Review by Mai-Vang51 — COMMENTED** [review]
Hi Salvin. Very sorry for the delay in reviewing your code. There's not much said but I commented on a couple of things in your project management as requested.

**Comment by Salvin-George** [comment]
all good :)

**Pull request merged**